src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerEngine.java

```diff
@@ -357,6 +334,13 @@ private void initialize(boolean createBamOutIndex, final boolean createBamOutMD5
                 : null);
     }
 
+    private HaplotypeCallerGenotypingEngine makeGenotypingEngine(int ploidy) {
```

Comment:
I pulled this out like you I was asked, I didn't pull out the one for the minimal genotyper because it relied on state from the function around it and it seemed messier to do so.  I'm hoping these can be simplified when you put in the copy methods.